### DISCUSSION TITLE
Limited documentation on array keys & pagination

### DISCUSSION THREAD

**Discussion by colinhacks** [vercel/swr#710] [Q&A] [answered]
Having read the full docs, I'm still confused about the behavior when using array keys.

- If any of the items in the array key are `null`, does SWR cancel the request?
- Do all items have to be strings? Another discussion seemed to imply that. If so, why isn't that enforced by the TypeScript typings?
- Pagination is listed as a feature but not documented at all?

I appreciate any help/clarification here.

**Comment by promer94** [comment, marked as answer]
1. No, swr will still fire the request
```js
const { data } = useSWR(['/api', null], (...args) => {
  console.log(...args) // api null
})
```
2. items in the array key could be object. But swr shallowly compares (===) the arguments on every render, and triggers revalidation if any of them has changed. [Arguments](https://swr.vercel.app/docs/arguments)
3. The docs does have a section about [Pagination](https://swr.vercel.app/docs/pagination)

**Comment by colinhacks** [comment]
Ah thanks! I didn't know there was additional documentation at swr.vercel.com. Seems like they tried to add all the docs into the README and gave up at some point. Strange.

Thanks for your help!
